Update PII guides with Data Protection Plus (#1109)
vtexdocs/dev-portal-content: docs/guides/Integration-Guides/data-privacy/limitations-of-the-pii-data-architecture-during-closed-beta.md

---
title: "Limitations of the PII data architecture during closed beta"
slug: "limitations-of-the-pii-data-architecture-during-closed-beta"
hidden: false
createdAt: "2022-02-22T22:29:28.541Z"
updatedAt: "2023-05-16T09:38:36.446Z"
seeAlso:
 - "/docs/guides/data-protection-plus"
 - "/docs/guides/pii-data-architecture-specifications"
 - "/docs/guides/data-residency"
 - "/docs/guides/profile-system"
 - "/docs/guides/changes-in-vtex-features-behavior-to-handle-pii-data"
---

>⚠️ [Data Protection Plus](https://developers.vtex.com/docs/guides/data-protection-plus) is in closed beta phase, only available in select regions.<br /><br />
> This feature is part of [VTEX Shield](https://help.vtex.com/en/tutorial/vtex-shield--2CVk6H9eY2CBtHjtDI7BFh). If you are already a VTEX customer and want to adopt VTEX Shield for your business, please contact [Commercial Support](https://help.vtex.com/en/tracks/support-at-vtex--4AXsGdGHqExp9ZkiNq9eMy/3KQWGgkPOwbFTPfBxL7YwZ). Additional fees may apply. If you are not yet a customer but are interested in this solution, please complete our [contact form](https://vtex.com/us-en/contact/).

> Some commerce features of the VTEX platform are not available for accounts using the PII data architecture while [Data Protection Plus](https://developers.vtex.com/docs/guides/data-protection-plus) is in closed beta phase, as we are working to evolve it and ensure data privacy best practices across the platform.

Reviewer: A frase ficou um pouco longa. Sugiro dividir para facilitar a leitura

```suggestion
Some commerce features of the VTEX platform are unavailable for accounts using the PII data architecture during the closed beta phase of [Data Protection Plus](https://developers.vtex.com/docs/guides/data-protection-plus). We are actively enhancing this feature to ensure the best data privacy practices across the platform.
```
